src/Factory.php

```diff
@@ -220,7 +220,7 @@ public static function setReaderPluginManager(ReaderPluginManager $readers)
     public static function getReaderPluginManager()
     {
         if (static::$readers === null) {
-            static::$readers = new ReaderPluginManager();
+            static::$readers = new ReaderPluginManager(new ServiceManager());
```

Comment:
This is required, as abstract plugin manager implementations now require the parent service manager during construction. I did the same with the writer plugin manager as well.